docs/OPENSHIFT_GUIDE.md

```diff
@@ -67,17 +67,18 @@ oc apply -f openshift/service-mesh/smcp.yaml
 oc wait --for=condition=ready pod --all -n openshift-operators --timeout=300s
 oc wait --for=condition=ready pod --all -n istio-system --timeout=300s
 
-# Make sure to add your namespaces to the ServiceMeshMemberRoll and create a
-# PeerAuthentication Policy for each of your namespaces
+# Make sure to add your namespaces to the ServiceMeshMemberRoll
 oc create ns kserve 
 oc create ns kserve-demo
 oc apply -f openshift/service-mesh/smmr.yaml
-oc apply -f openshift/service-mesh/peer-authentication.yaml
 
 # Install OpenShift Serverless operator
 oc apply -f openshift/serverless/operator.yaml
 oc wait --for=condition=ready pod --all -n openshift-serverless --timeout=300s
```

Comment:
Why don't you move these lines above line #70 like following?
~~~
# Install OpenShift Serverless operator
oc apply -f openshift/serverless/operator.yaml
oc wait --for=condition=ready pod --all -n openshift-serverless --timeout=300s

# Make sure to add your namespaces to the ServiceMeshMemberRoll and create a
# PeerAuthentication Policy for each of your namespaces
oc create ns kserve         
oc create ns kserve-demo
oc apply -f openshift/service-mesh/smmr.yaml
oc apply -f openshift/service-mesh/peer-authentication.yaml
~~~

Either way is ok to me but the typical order to set kserve serverless mode dependencies is the following
- Install/configure ISTIO
- Install/configure Knative
- Configure the rest of the setup such as smmr, peer-authencation and so on.
- Install/configure Kserve